turbo/engineapi/engine_types/jsonrpc.go

```diff
@@ -68,6 +69,24 @@ type BlobsBundleV1 struct {
 	Blobs       []hexutility.Bytes `json:"blobs"       gencodec:"required"`
 }
 
+func (b *BlobsBundleV1) Check() error {
```

Comment:
Why is this needed?

Reply:
just simply check if content length is reasonable or not. maybe move it out from structure definition.